apps/files_external/lib/Lib/Backend/SMB.php

```diff
@@ -53,6 +53,8 @@ public function __construct(IL10N $l, Password $legacyAuth) {
 				(new DefinitionParameter('show_hidden', $l->t('Show hidden files')))
 					->setType(DefinitionParameter::VALUE_BOOLEAN)
 					->setFlag(DefinitionParameter::FLAG_OPTIONAL),
+				(new DefinitionParameter('timeout', $l->t('Timeout')))
+					->setFlag(DefinitionParameter::VALUE_HIDDEN)
```

Comment:
```suggestion
					->setType(DefinitionParameter::VALUE_HIDDEN)
```

Guessing from https://github.com/nextcloud/server/blob/c43e0d6e283773b5c531d55fbd652879849b9f4d/apps/files_external/templates/settings.php#L73 VALUE_HIDDEN should be the type.

Reply:
:see_no_evil: